nautobot/project-static/css/dark.css

```diff
@@ -11,19 +11,19 @@ html[data-theme="dark"] img {
 }
 /* Makes badge text white in dark mode */
 html[data-theme="dark"] .badge {
-  color: black;
+  filter: invert(1) hue-rotate(180deg);
 }
 /* White text on buttons */
 html[data-theme="dark"] button {
-  color: black;
+  filter: invert(1) hue-rotate(180deg);
 }
 /* White text on buttons */
 html[data-theme="dark"] .btn {
-  color: black;
+  filter: invert(1) hue-rotate(180deg);
 }
 /* White text on labels */
 html[data-theme="dark"] .label {
-  color: black !important;
+  filter: invert(1) hue-rotate(180deg);
 }
```

Comment:
Can we change the invert for images above to:

`html[data-theme="dark"] img, html[data-theme="dark"] .badge, html[data-theme="dark"] button, ...` to capture them all with one statement?